examples/defer/defer.go

```diff
@@ -40,5 +40,9 @@ func writeFile(f *os.File) {
 
 func closeFile(f *os.File) {
 	fmt.Println("closing")
-	f.Close()
+	err := f.Close()
+	if err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
The `_,_` here doesn't seem necessary. Just call `fmt.Fprintf` without assigning return values

Reply:
It's not necessary, but not explicitly carrying about the return values, has lead to this very "bug".
But surely it will run without it.

Reply:
I wouldn't call it a bug, more like brevity for example. but since we have the separate `closeFile` function anyway, I feel it's OK to be more explicit here.

Regardless, `_, _ = ...` is not idiomatic in Go.